reference/src/packages.md

```diff
@@ -382,16 +371,15 @@ named_addr = "0xC0FFEE"
 ## Usage and Artifacts
 
 The Move package system comes with a command line option as part of the CLI:
-`sui move <command> <command_flags>`. Unless a particular path is provided, all
-package commands will run in the current working directory. The full list of
-commands and flags for the Move CLI can be found by running `sui move --help`.
+`sui move <command> <command_flags>`. Unless a particular path is provided, all package commands
+will run in the current working directory. The full list of commands and flags for the Move CLI can
+be found by running `sui move --help`.
```

Comment:
This seems a little unclear, because, e.g., `sui move test` will work at any directory in a package.

Reply:
Yea this is a great callout -- this should be something like (or similar)
```
... Unless a particular path is provided, all package commands will run in the current enclosing Move package. ....
```